app/views/admin/users/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Profile</h1>
+
+<h2>Welcome <%= @user.name %>!</h2>
```

Comment:
Should this still say welcome even when it is an admin looking at other users?

Reply:
Maybe change it to `Viewing <%= @user.name %>'s profile`? or something to that extent